0.wave1_data/3.process_features/3.process_features.yml

```diff
@@ -0,0 +1,18 @@
+name: 3.process_features_interstellar
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.8.13
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::jupyterlab
+  - conda-forge::pandas=1.4.2
+  - conda-forge::plotnine
+  - conda-forge::ipykernel
+  - conda-forge::nb_conda_kernels
+  - conda-forge::jupyter_contrib_nbextensions
+  - conda-forge::scipy=1.7
+  - conda-forge::numpy=1.22
```

Comment:
Consider updating to the latest version of numpy unless there are specific reasons for including this version. If there are specific reasons, consider leaving a comment as to why. This applies to other static dependencies in this file as well.

Reply:
I mainly have it this way because this is what Greg did and I knew it would work.

@gwaybio Is there any reason to have them static?